activestorage/test/models/blob_test.rb

```diff
@@ -43,6 +43,21 @@ class UserWithHasOneAttachedDependentFalse < User
     assert_equal "text/plain", blob.content_type
   end
 
+  test "create after upload extracts content_type from io when no content_type given" do
+    blob = create_csv_blob content_type: nil
+    assert_equal "text/csv", blob.content_type
+  end
```

Comment:
This test doesn’t seem terribly valuable. I’d omit it.

Reply:
Modified for content_type nil and identify: false e34eec2c2c2133edf4512b1370455fe4c669c0ca